mkosi.profiles/obs/debian.repo/systemd-opensuse.sources

```diff
@@ -0,0 +1,45 @@
+Types: deb
```

Comment:
Please just name this file debian.sources and get rid of the debian.repo directory